runtime/gc_glue_java/ConfigurationDelegate.hpp

```diff
@@ -302,7 +302,7 @@ class MM_ConfigurationDelegate
 
 		extensions->accessBarrier->initializeForNewThread(env);
 
-		if (extensions->isConcurrentMarkEnabled()) {
+		if ((extensions->isConcurrentMarkEnabled()) && (!extensions->usingSATBBarrier())) {
```

Comment:
We have isIncrementalUpdateBarrierEnabled in OMR Config class, that might be visible to use?

We also have usingSATBBarrier wrapper in Ext class (for isSnapshotAtTheBeginningBarrierEnabled in Conf class).
Perhaps we should introduce similar wrapper for IncUpdate in Ext (for symmetry) and use the wrapper (here and possible elsewhere)? Obviously it would be later, since requires more OMR changes...

Reply:
> We have isIncrementalUpdateBarrierEnabled in OMR Config class, that might be visible to use?

A wrapper in Ext will work. I had issues with including Configuration class header `#include "Configuration.hpp"` here  I tried doing something like `extensions->configuration->isIncrementalUpdateBarrierEnabled` didn't work, I think it'll need some changes to build/make files.